src/Lecture2.hs

```diff
@@ -191,7 +208,13 @@ verify that.
 [1,2,3,4,7]
 -}
 merge :: [Int] -> [Int] -> [Int]
-merge = error "TODO"
+merge [] [] = []
+merge x []  = x
+merge [] x  = x
+merge (x : xs) (y : ys) =
+  if x < y
+  then x : merge xs (y : ys)
+  else y : merge (x : xs) ys
```

Comment:
Absolutely correct solution! 💎 
You can make it more performant by checking if both heads of the given lists are equal. In that case, you're able to reduce the size of each list by 1 on each recursive call instead of reducing the size of only a single list ✂️